server/planning/feeding_services/onclusive_api_service.py

```diff
@@ -80,8 +82,8 @@ def _update(self, provider, update):
             end_date = current_date + timedelta(days=int(provider["config"]["days_to_ingest"]))
             for offset in range(100, ONCLUSIVE_MAX_OFFSET, 1000):
```

Comment:
since you have that LIMIT defined now you can use it here as the 3rd param so it's in sync as it should be